### PR TITLE
feat(config): Add config and config describe cmds

### PR DESCRIPTION
**Pull request by raulb**
# Description of change

Fixes https://github.com/meroxa/product/issues/103

This pull-request adds two new commands:

- `meroxa config` (which is aliased as `meroxa cfg`)
- `meroxa config describe` (or `meroxa cfg describe`).

For now, `meroxa config` executes `meroxa config describe` since at the moment, we are not adding new functionality to the `config` command. Eventually, we'd like to give the ability to update, etc our CLI configuration.

Notice this pull-request is also renaming `meroxa env` to `meroxa config`. The main difference is that `meroxa env` wasn't documented whereas `meroxa config` will be documented (https://github.com/meroxa/meroxa-docs/pull/31).

Another thing that's worth mentioning is that we already had a global flag `--config` that currently lets users specify a different configuration file so it's possible someone could run a command such as:

```
meroxa config describe --config my-other-cfg-file-path
```

This new `config` commands plays nicely (it's consistent) with the use of that flag in other commands.

e.g.:

```
meroxa resources ls --config my-other-cfg-file-path
```

This pull-request also fixes how the documentation for our docs.meroxa.com is generated. Location changed in https://github.com/meroxa/meroxa-docs without updating the generation script here.

# Type of change

- [x]  New feature
- [x]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

# How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging

# Demo

**Before this pull-request**

```
❯ meroxa help
The Meroxa CLI allows quick and easy access to the Meroxa Data Platform.

Using the CLI you are able to create and manage sophisticated data pipelines
with only a few simple commands. You can get started by listing the supported
resource types:

meroxa resources list --types

Usage:
  meroxa [command]

Available Commands:
  api         Invoke Meroxa API
  auth        Authentication commands for Meroxa
  billing     Open your billing page in a web browser
  completion  Generate completion script
  connect     Connect two resources together
  connectors  Manage connectors on Meroxa
  endpoints   Manage endpoints on Meroxa
  help        Help about any command
  login       Login or Sign up to the Meroxa Platform
  logout      Clears local login credentials of the Meroxa Platform
  open        Open in a web browser
  pipelines   Manage pipelines on Meroxa
  resources   Manage resources on Meroxa
  transforms  Manage transforms on Meroxa
  version     Display the Meroxa CLI version
  whoami      Display the current logged in user


Flags:
      --config string      config file
      --debug              display any debugging information
  -h, --help               help for meroxa
      --json               output json
      --timeout duration   set the client timeout (default 10s)

Use "meroxa [command] --help" for more information about a command.

❯ meroxa help env
Show Meroxa CLI environment details

Usage:
  meroxa env [flags]

Flags:
  -h, --help   help for env

Global Flags:
      --config string      config file
      --debug              display any debugging information
      --json               output json
      --timeout duration   set the client timeout (default 10s)

❯ meroxa env
Using meroxa config located in "path"

access_token: 1234...890
actor: user@email.com
actor_uuid: my-uuid
refresh_token: foo...bar
```

**After this pull-request**

```
❯ .m help
The Meroxa CLI allows quick and easy access to the Meroxa Data Platform.

Using the CLI you are able to create and manage sophisticated data pipelines
with only a few simple commands. You can get started by listing the supported
resource types:

meroxa resources list --types

Usage:
  meroxa [command]

Available Commands:
  ...
  config      Manage your Meroxa CLI configuration
  ...

❯ .m help config
Manage your Meroxa CLI configuration

Usage:
  meroxa config [flags]
  meroxa config [command]

Aliases:
  config, cfg

Available Commands:
  describe    Show Meroxa CLI configuration details

Flags:
  -h, --help   help for config

Global Flags:
      --config string      config file
      --debug              display any debugging information
      --json               output json
      --timeout duration   set the client timeout (default 10s)

Use "meroxa config [command] --help" for more information about a command.

❯ .m config
Using meroxa config located in "path"

access_token: 1234...890
actor: user@email.com
actor_uuid: my-uuid
refresh_token: foo...bar

❯ .m config describe
Using meroxa config located in "path"

access_token: 1234...890
actor: user@email.com
actor_uuid: my-uuid
refresh_token: foo...bar
```

# Documentation updated

- [x]  https://github.com/meroxa/meroxa-docs/pull/31

### CODE REVIEW

File: docs/cmd/md/meroxa_config.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  -h, --help   help for config
+```
+
+### Options inherited from parent commands
+
+```
+      --config string      config file
+      --debug              display any debugging information
+      --json               output json
+      --timeout duration   set the client timeout (default 10s)
```
Comment (jayjayjpg) [inline docs/cmd/md/meroxa_config.md:21]: I'm unsure if there's a specific format the docs usually follow, but would it also make sense to do something like the following?

```suggestion
      --timeout number  set the duration of the client timeout in seconds (default 10s)
```

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I left one more question on the docs, but otherwise this looks good to go!

File: docs/cmd/md/meroxa_config.md (hunk 1 above)
Comment (raulb) [reply docs/cmd/md/meroxa_config.md:21]: @jayjayjpg that's a nice suggestion, thank you. I'll change it [in the implementation](https://github.com/meroxa/cli/blob/master/cmd/meroxa/global/global.go#L50) (docs are autogenerated)

**Review by dianadoherty — APPROVED** [review]

**Comment by raulb** [comment]
Shipping this pull-request, although we won't create a release of 1.1.0 until Tuesday 24th ([see comment](https://github.com/meroxa/meroxa-docs/pull/31#pullrequestreview-734216632)).

**Pull request merged**